set.mm

```diff
@@ -438942,6 +438942,9 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "E**" as "E**";
   althtmldef "E**" as '&exist;**';
   latexdef "E**" as "\exists^{**}";
+htmldef "F//" as "F//";
```

Comment:
(Non-blocking) I still think you should introduce global variables with your personal prefix, as I did with wl-in, to avoid clashes in future, and discussions as in #3301.

Reply:
I think that `F//` is weird enough to avoid clashes (in the representation, I dit put a "prime" to differentiate it from the symbol for `F/`. If there is a clash in the future, then the desired token/symbol should be used for the main part, and the ones in the mathbox changed.